frontend/src/pages/modelRegistry/screens/RegisteredModels/usePrefillDeployModalFromModelRegistry.ts

```diff
@@ -20,13 +20,12 @@ const usePrefillDeployModalFromModelRegistry = (
   registeredModelDeployInfo?: RegisteredModelDeployInfo,
 ): [LabeledConnection[], boolean, Error | undefined] => {
   const [fetchedConnections, connectionsLoaded, connectionsLoadError] = useConnections(
-    projectContext ? undefined : createData.project,
+    projectContext ? projectContext.currentProject.metadata.name : createData.project,
```

Comment:
@manaswinidas @mturley we can wait to merge this until Manaswini makes her changes or if you want to test MR with this PR that works too.

Reply:
I can test this with MR once you rebase.

Reply:
This change doesn't affect Model registry prefilling. No regressions here.